apps/widget-configurator/src/common/analytics/types.ts

```diff
@@ -0,0 +1,8 @@
+import { GtmEvent } from '@cowprotocol/analytics'
+
+export enum WidgetCategory {
```

Comment:
I would suggest to rename the enum in order to give more context: `AnalyticsCategory`

Reply:
Addressed